tests/ui/sized/sized-reference-to-unsized.rs

```diff
@@ -1,9 +1,12 @@
+//! Check that a reference to a potentially unsized type (`&T`) is itself considered `Sized`.
+//! This allows functions requiring `Sized` types to accept references to dynamically sized types.
```

Comment:
```suggestion
```

Also redundant with the first line (`&T` as `Sized` allows more behavior than just functions anyway)